search_blackbox_test.go

```diff
@@ -28,6 +30,16 @@ import (
 	"golang.org/x/net/context"
 )
 
+var spaceBlackBoxTestConfiguration *config.ConfigurationData
```

Comment:
Hey @hectorj2f 
Thanks so much for the suggestion - keep 'em coming! 

I tried this out  in a couple of test files under login/service_*****_test.go
Worked fine..

But the stuff inside ./login_test.go is failing to access the initialized object

```
ERRO[0000] Unable to get Keycloak auth endpoint URL      err="Bad value for parameter 'host': '' (expected: 'must contain more than one domain')" file="/Users/shoubhikbose/dev/alm-core-sbose/src/github.com/almighty/almighty-core/login.go" func="github.com/almighty/almighty-core.(*LoginController).Authorize" line=57 pid=56505 pkg="github.com/almighty/almighty-core" req_id=
```

I might need to dig up further why that's happening. Would take me a while to make changes to 20 odd files after that :( 

Do you mind if I postpone this change to a different PR , because it looks like I might have to spend significant time on getting this fixed ?